Assets/Mirror/Components/NetworkTransform2k/NetworkTransformBase.cs

```diff
@@ -446,7 +446,7 @@ protected virtual void OnValidate()
             // make sure that catchup threshold is > buffer multiplier.
             // for a buffer multiplier of '3', we usually have at _least_ 3
             // buffered snapshots. often 4-5 even.
-            catchupThreshold = Mathf.Max(bufferTimeMultiplier + 1, catchupThreshold);
+            catchupThreshold = Mathf.Max(bufferTimeMultiplier + 2, catchupThreshold, 4);
```

Comment:
please add comments.
it won't be obvious why +2 or ',4' in 5 years from now :)

Reply:
Added commentary.

Reply:
Simplified validation to bufferTimeMultiplier + 3 to cover for calculation of snapshots in excess of the first 2, and also having old enough snapshots to cover for buffer after the 3rd snapshot.